src/algebra/algebra/operations.lean

```diff
@@ -164,6 +164,46 @@ calc map f.to_linear_map (M * N)
       simp [fy_eq] }
   end
 
+open_locale classical
+
+namespace finset
+/-- A finite set `U` contained in the product of two sets `S * S'` is also contained in the product
+of two finite sets `T * T' ⊆ S * S'`. -/
+lemma subset_mul {M : Type*} [monoid M] {S : set M} {S' : set M} {U : finset M} (f : ↑U ⊆ S * S') :
```

Comment:
I think this lemma should now move to a file about pointwise multiplication (`algebra/pointwise1, I think?)

Reply:
Also, if you add the `@[to_additive]` attribute, you'll have `subset_add` automatically.

Reply:
I see, thank you! I have moved the lemma.